bedrock/mozorg/templates/mozorg/moss/index.html

```diff
@@ -0,0 +1,292 @@
+      {% endtrans %}
+
+        {{ _('Suggest a recipient.') }}
+      </p>
+
+      <div class="moss-ctas">
+        <p>
+          <a href="{{ url('mozorg.moss.secure-open-source') }}" class="moss-copy">{{ _('Learn More') }}</a>
+        </p>
+
+        <p>
+          <a href="https://docs.google.com/a/mozilla.com/forms/d/e/1FAIpQLScLwANEOvLBE6gnFVoiamqHOYzzkaChpdQJ7f0PlZGmfyy94w/viewform" rel="external" class="button">{{ _('Nominate Now') }}</a>
+        </p>
+      </div>
+    </li>
+  </ul>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('A few details') }}</h2>
+
+  <p class="moss-copy">
+    {{ _('MOSS applications are considered in batches every three months.') }}
+
+  {% trans %}
+    Missing a deadline may cause a delay in the consideration of your
+    application.
+  {% endtrans %}
+
+  {% trans url1='https://wiki.mozilla.org/MOSS#Application_Deadlines' %}
+    Information on the next deadline can be found
+    <a href="{{ url1 }}">on the MOSS wiki</a>.
+  {% endtrans %}
+  </p>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('Who decides?') }}</h2>
+
+  <p class="moss-copy">
+  {% trans %}
+    We have formed a selection committee to assess awards on the Foundation
+    Technology and Mission Partners tracks.
+  {% endtrans %}
+
+  {% trans %}
+    It is comprised partly of current, committed Mozillians — they bring a good
+    working knowledge of Mozilla’s day- to-day activities and how various open
+    source projects are used.
+  {% endtrans %}
+
+  {% trans %}
+    There are also senior Mozilla Alumni — they are no longer actively involved
+    with Mozilla on a day-to-day basis but have a deep understanding of our
+    project and a different/outside perspective.
+  {% endtrans %}
+
+  {% trans %}
+    Finally, there are other open source experts — they bring knowledge of the
+    role of different projects within the open source ecosystem.
+  {% endtrans %}
+  </p>
+
+  <p class="moss-ctas">
+    <a href="https://wiki.mozilla.org/MOSS#Selection_Committee" rel="external" class="moss-copy">{{ _('Meet the committee') }}</a>
+  </p>
+</section>
+
+<section class="moss-section">
+  <h2 class="moss-section-heading">{{ _('Past Recipients') }}</h2>
+
+  <p class="moss-copy">
+  {% trans %}
+    Check out a few of the Mozilla Open Source Support past recipients.
+  {% endtrans %}
+  </p>
+
+  <ul class="moss-list moss-list-thirds">
+    <li>
+      {{ high_res_img('mozorg/moss/tor.png', {'alt': 'Tor', 'width': '300', 'height': '169'}) }}
+
+      <h3 class="moss-subhead">{{ _('$%(sum)s')|format(sum='152,500') }}</h3>
+
+      <p class="moss-copy">
+      {% trans %}
+        Tor is free software and an open network that helps you defend against
+        network surveillance.
+      {% endtrans %}
+      </p>
+
+      <p class="moss-ctas">
+        <a href="https://www.torproject.org/" rel="external" class="moss-copy">https://www.torproject.org</a>
```

Comment:
Do these links really need to display the protocol? I think just "torproject.org" would be better for readability. I suppose this is a creative/copy decision but showing protocols here seems clunky and unnecessary. If the issue was already raised and this is what they want, so be it, but if it wasn't raised before I think we should ask now.